sample/src/main/res/layout/activity_main.xml

```diff
@@ -22,6 +22,7 @@
         app:dayBackgroundColor="#05000000"
         app:todayBackgroundColor="#1848adff"
         app:headerColumnBackground="#ffffffff"
-        app:todayHeaderTextColor="@color/accent"/>
+        app:todayHeaderTextColor="@color/accent"
+        app:showHalfHours="false"/>
```

Comment:
Thank you so much for your reply. I already have changed all you suggest, and i thought it would work properly, but it isn't so.  These are the steps i done: Change DateTime Interpreter like so:
String interpretTime(int hour, int minutes); Then i have copied enterely the mainactivity above and added app:showHalfHours:="true" in my activity_main.xml. However It doesn't works. The string i have posted above gives error yet.